strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java

```diff
@@ -42,6 +43,29 @@
     @Inject
     ArtifactEntryService artifactEntryService;
 
+
+
+    @Test
+    public void saveEntityShouldWork()
+        throws Exception
+    {
+        ArtifactEntry artifactEntry = new ArtifactEntry();
+        artifactEntry.setStorageId(storageId);
+        artifactEntry.setRepositoryId(repositoryId);
+        artifactEntry.setArtifactCoordinates(new NullArtifactCoordinates(String.format("%s/%s/%s/%s", groupId, artifactId + "123", "1.2.3", "jar")));
+        assertThat(artifactEntry.getCreated(), CoreMatchers.nullValue());
```

Comment:
Cosmetic: Could you please keep `assert*`, `logger.info/debug/etc` as separate blocks like this:

```
    String s = getFoo();

    logger.debug("Got foo");
    logger.debug("foo = %s");

    blah();

    assertNotNull(s);
    assertEquals("blah", s);
```